spinalcordtoolbox/scripts/sct_propseg.py

```diff
@@ -611,6 +610,7 @@ def propseg(img_input, options_dict):
     # If using OptiC
     elif use_optic:
         image_centerline = optic.detect_centerline(image_input, contrast_type, remove_temp_files)
+        path_tmp = tmp_create(basename="propseg-centerline-optic")
```

Comment:
This isn't a new issue, but I'm noticing it now, so I might as well comment: I'm a little concerned that this call to `tmp_create()` doesn't have any corresponding logic to remove it if `remove_temp_files` is set. That means that `sct_propseg` leaves the generated centerline lying around, even after it's done using it.

But, as you noted in your commit message, fixing this would be out of scope for this PR.